docs/book/src/developer/providers/v1.0-to-v1.1.md

```diff
@@ -0,0 +1,46 @@
+# Cluster API v1.0 compared to v1.1
+
+This document provides an overview over relevant changes between ClusterAPI v1.0 and v1.1 for
+maintainer of other providers and consumers of our Go API.
+
+## Minimum Go version
+
+- The Go version used by Cluster API is still Go 1.16+
+
+## Dependencies
+
+**Note**: Only the most relevant dependencies are listed, `k8s.io/` dependencies
+are kept in sync with the versions used by `sigs.k8s.io/controller-runtime`. 
+
+- sigs.k8s.io/controller-runtime: v0.10.x => v0.10.x
+- sigs.k8s.io/controller-tools: v0.7.x => v0.7.x
+- sigs.k8s.io/kind: v0.11.x => v0.11.x
+
+## Changes by Kind
+
+### Deprecation
+
+* The Cluster and ClusterClass webhooks have been moved to the `webhooks` package. Thus, the following methods on `Cluster` and `ClusterClass` 
+  in `api/v1beta1` are deprecated: `SetupWebhookWithManager`, `Default`, `ValidateCreate`, `ValidateUpdate` and `ValidateDelete`.
+* The `third_party/kubernetes-drain` package is deprecated, as we're now using `k8s.io/kubectl/pkg/drain` instead ([PR](https://github.com/kubernetes-sigs/cluster-api/pull/5440)). 
+* `util/version.CompareWithBuildIdentifiers` has been deprecated, please use `util/version.Compare(a, b, WithBuildTags())` instead.
+
+### API Change
+
+* Some controllers have been moved to internal to reduce there API surface. We now only
+  surface what is necessary, e.g. the reconciler and the `SetupWithManager` func:
+    * [bootstrap/kubeadm](https://github.com/kubernetes-sigs/cluster-api/pull/5493) 
+
+### Other
+
+* ClusterClass:
```

Comment:
Trying to ease adoption a bit here. The link is broken pending that the corresponding PR is merged.

Reply:
Which link is broken here? I think they're working for me

Reply:
Ah nice, only the anchor is broken :)
https://cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#clusterclass-definitions

Reply:
That's one way to work around the link checker :D